src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/AspNetCoreInstrumentationEventSource.cs

```diff
@@ -43,16 +43,16 @@ public void NullPayload(string handlerName, string eventName)
             this.WriteEvent(1, handlerName, eventName);
         }
 
-        [Event(2, Message = "Request is filtered out.", Level = EventLevel.Verbose)]
-        public void RequestIsFilteredOut(string eventName)
+        [Event(2, Message = "Request is filtered out from handler '{0}'.", Level = EventLevel.Verbose)]
+        public void RequestIsFilteredOut(string handlerName, string eventName)
```

Comment:
I didn't bump the event versions. It doesn't seem like we care about ETW consumers that cache manifests.